js/clients/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@summa-tx/bitcoin-spv-js-clients",
+  "version": "0.1.0",
+  "description": "Clients for fetching bitcoin SPV proofs in Javascript",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">10.0.0"
+  },
+  "engineStrict": true,
+  "scripts": {
+    "lint": "eslint ./test ./src",
+    "lint:fix": "eslint --fix ./test ./src",
```

Comment:
these don't lint anything in this package

Reply:
Fixed